### PR TITLE
automatically add Test suffix to tests generated through make:test command

### PR DESCRIPTION
**Pull request by mstnorris**

### CODE REVIEW

**Comment by fernandobandeira** [comment]
:+1: 

Since the 'Test' suffix is required there's no harm in adding it by default when creating a Test,

File: src/Illuminate/Foundation/Console/TestMakeCommand.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ protected function getPath($name)
     {
         $name = str_replace($this->laravel->getNamespace(), '', $name);
 
-        return $this->laravel['path.base'].'/tests/'.str_replace('\\', '/', $name).'.php';
+        return $this->laravel['path.base'].'/tests/'.str_replace('\\', '/', $name).'Test.php';
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/TestMakeCommand.php]: We probably need to remove Test from the end if it's there too.

Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/TestMakeCommand.php]: Same for generating the class name

Comment (fernandobandeira) [inline src/Illuminate/Foundation/Console/TestMakeCommand.php]: Yep, a ``` trim($name, 'Test') ``` or something similar should do the trick

**Comment by mstnorris** [comment]
@GrahamCampbell you could do that, but, that could be overkill as when someone pull's in Jeffrey Way's Generators, the "TableSeeder" suffix is automatically applied, and if the user already adds it themselves then it is just duplicated. So the user learns not to use it again.

Either way, a simple `str_replace` would do it.

**Comment by fernandobandeira** [comment]
Yes or something like

```php
preg_replace('/Test$/', '', $str);
```

Because str_replace would cut something like ```TestingThis```

**Comment by GrahamCampbell** [comment]
Well, your PR is broken right now because it doesn't match the filename to the classname.

**Comment by mstnorris** [comment]
Would adding this to the `getPath` method?

```
preg_replace('/Test$/', '', $name);
```

**Comment by taylorotwell** [comment]
Will just prefer to be explicit here.

**Pull request closed without merging**